cmake/config.cmake

```diff
@@ -62,6 +62,12 @@ endif()
 ## Do NOT force it since it is explicitly disabled on Windows for now.
 if(OMR_JITBUILDER)
 	set(OMR_JITBUILDER_TEST ON CACHE BOOL "")
+
+	## Enable additional JitBuilder tests if running on a supported platform
+	if(OMR_HOST_ARCH STREQUAL "x86" AND OMR_TEMP_DATA_SIZE STREQUAL "64")
+    		set(OMR_JITBUILDER_ADDITIONAL ON)
```

Comment:
I am not sure this works on Windows

Reply:
Also since we are making changes here could we change the name of this flag to something like OMR_JITBUILDER_TEST_EXTENDED or something similar so it is more clear that it is controlling tests and not JITBuilder features?